controllers/errors.js

```diff
@@ -0,0 +1,10 @@
+exports.wrongPaths = (req, res, next) => {
+    res.status(404).send({ msg: "Wrong URL!" });
+};
+
+
+exports.catchAll = (err, req, res, next) => {
+    if (err.code === '22P02') {
```

Comment:
Our catch all should just be for 500s. At this point we don't need to handle our psql errors, so this logic shouldn't be here yet, but also i would suggest this logic should (at a later point when it is needed) be in its own function and have it's own position in our error handling chain